core/sql/common/ComMisc.cpp

```diff
@@ -99,3 +99,17 @@ NABoolean ComIsTrafodionReservedSchemaName(
   return FALSE;
 }
 
+// schema names of pattern "_HV ... _" and "_HB_ ... _" are reserved to store
+// external hive and hbase tables
+NABoolean ComIsTrafodionExternalSchemaName (
+                                    const NAString &schName)
+{
+  Int32 len (schName.length());
+  Int32 prefixLen = sizeof(HIVE_EXT_SCHEMA_PREFIX);
```

Comment:
the prefixes are the same but should probably fix this in case they change or we add something different.